crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -1,46 +1,48 @@
-use conversions::{byte_array::ByteArray, felt::FromShortString};
+use conversions::byte_array::ByteArray;
 use regex::Regex;
 use starknet_types_core::felt::Felt;
 
 #[must_use]
 pub fn try_extract_panic_data(err: &str) -> Option<Vec<Felt>> {
-    // FIXME add support for comma separated errors
+    // Regex used to extract panic data from panicking proxy contract
     let re_felt_array = Regex::new(
```

Comment:
? Or something similar?

```suggestion
    let re_error_prefix = Regex::new(
```